static/app/components/commandPalette/useCommandPaletteActions.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: useCommandPaletteActions
```

Comment:
Would love to hear any feedback about writing mdx stories. The toolchain isn't perfect, and if there's some stuff that was particularly annoying, we'd be happy to improve it

Reply:
It was pretty seamless once I set up my editor to format and synax highlight the mdx files.

One thing I noticed was that the stories sidebar assumes everything is a component (casing): <img width="263" height="81" alt="CleanShot 2025-10-20 at 15 42 37" src="https://github.com/user-attachments/assets/53e38e33-1096-4c3f-839a-e7b47aecfd40" />

Reply:
They are not... It's the biggest downside of MDX as things stand, and has been asked for by many folks already. We plan to look into supporting it once UI2 is released